webpack.config.js

```diff
@@ -81,6 +81,8 @@ module.exports = {
               "@babel/plugin-transform-flow-strip-types",
               "@babel/plugin-transform-react-display-name",
               "@babel/plugin-proposal-class-properties",
+              "@babel/plugin-proposal-logical-assignment-operators",
+              "@babel/plugin-proposal-nullish-coalescing-operator",
```

Comment:
as a bonus, I've also added this plugin since it's like a sibling feature to the optional chaining (were introduced at the same time) and you had that one already here

Reply:
Yeah, that sounds like a good idea.

Reply:
Yes, the nullish coalescing operator would have been the next syntax error :) good catch!